packages/trie/package.json

```diff
@@ -61,7 +61,6 @@
   },
   "devDependencies": {
     "@ethereumjs/genesis": "0.1.0-rc.1",
-    "0x": "^4.9.1",
```

Comment:
I am personally ok with dropping the `0x` dependency, this was added for convenience I guess when we were very much into "flame-graphing" 😂, but I guess it's totally ok respectively likely better if such a tools are just installed in a global scope by everyone who uses them.

Reply:
it wasn't in use anywhere that i could find :shrug: 

and it was responsible for like 9 vulnerabilities :laughing: